integration/business-process-tests/tests-integration/bpel/src/test/java/org/wso2/ei/businessprocess/integration/tests/bpel/upload/BpelVersioningTest.java

```diff
@@ -140,5 +155,30 @@ public void cleanup() throws PackageManagementException, InterruptedException, R
         this.loginLogoutClient.logout();
     }
 
+    private Callable<Boolean> isInforListAvailable(final String packageName) {
+        return new Callable<Boolean>() {
+            @Override
+            public Boolean call() throws Exception {
+                if (bpelProcessManagementClient.getProcessInfoList(packageName) != null) {
+                    return true;
+                } else {
+                    return false;
+                }
+            }
+        };
+    }
+
+    private Callable<Boolean> isProcessRetired() {
+        return new Callable<Boolean>() {
+            @Override
+            public Boolean call() throws Exception {
+                if (bpelProcessManagementClient.getStatus(activeStatus.get(0)).equals(ProcessStatus.RETIRED.toString())) {
+                    return true;
+                } else {
+                    return false;
```

Comment:
we do not nee else block, just return false after if block

Reply:
Fixed. Please refer latest commit.